gnomad_qc/v4/assessment/calculate_per_sample_stats.py

```diff
@@ -479,26 +529,49 @@ def main(args):
                 vep_canonical=args.vep_canonical,
                 vep_mane=args.vep_mane,
                 rare_variants_afs=rare_variants_afs,
+            ).write(filtering_groups_res.path, overwrite=overwrite)
+
+        if args.create_per_sample_counts_ht:
+            logger.info(
+                "Calculating per-sample variant statistics for %s...", data_type
             )
-            create_per_sample_counts_ht(mt, filter_groups_ht).write(
+            if test:
+                logger.warning(
+                    "This test requires that the full filtering groups Table has been "
```

Comment:
If it's a full filtering groups Table, then in assessment and above output path, do you still need to put "test"?

Reply:
Because if I want to run the next steps, with the written table, I don't have variant_ac in it.